index.html

```diff
@@ -29,54 +28,15 @@ <h2 id="project_tagline">Next-generation spectral-timing software</h2>
-Once you've familiarized yourself with the basics of Stingray, go to the <a alt="Stingray: issues" href="https://github.com/stingraysoftware/stingray/issues">Stingray issues page</a> and try to tackle one!
-Other ways to get involved are outlined on the <a alt="Timelab project ideas" href="http://timelabtechnologies.com/ideas.html">project ideas page</a>, along with some astrophysical background/motivation.</p>
-<p>For organizing and coordinating the software development, we have a Slack workspace -- please use <a alt="Join Stingray on Slack" href="https://stingray-slack.herokuapp.com/">this link for Slack</a> or send <a alt="Stingray: people" href="https://github.com/orgs/StingraySoftware/people">one of us</a> an email to join. Students interested in Stingray projects for Google Summer of Code, please join our Slack and check out the pinned posts!</p>
-
-<h4><a id="citing-stingray" class="anchor" href="#citing-stingray" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Citing Stingray</h4>
-<p>If you use Stingray, <em>please cite us in papers and other projects!</em>
-Citations are still the main currency of the academic world, and pointing to citations is <em>the</em> best way to help us ensure that Stingray continues to be supported and we can continue working on it!
-<br/><a alt="Stingray paper" href="https://ui.adsabs.harvard.edu/abs/2019ApJ...881...39H/abstract">Huppenkothen et al. 2019, ApJ 881, p. 39</a>
-<br/><a href="http://ascl.net/1608.001"><img src="https://img.shields.io/badge/ascl-1608.001-blue.svg?colorB=262255" alt="ascl:1608.001" /></a></p>
-
-<h4><a id="presentations" class="anchor" href="#presentations" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Presentations</h4>
-<p><ul>
-    <li>(upcoming) AAS 17th High-Energy Astrophysics Division meeting 2019</li>
-    <li><a alt="Stingray EWASS2018" href="http://ascl.net/wordpress/2018/05/24/software-in-astronomy-symposium-presentations-part-3/">European Week of Astronomy and Space Science 2018</a></li>
-    <li><a alt="Stingray HEAD17" href="https://speakerdeck.com/abigailstev/stingray-open-source-spectral-timing-software">AAS 16th High-Energy Astrophysics Division meeting 2017</a></li>
-    <li><a alt="Stingray EWASS17" href="http://ascl.net/wordpress/2017/07/23/special-session-on-and-about-software-at-ewass-2017/">European Week of Astronomy and Space Science 2017</a></li>
-    <li><a alt="Stingray PyAstro16" href="https://speakerdeck.com/abigailstev/stingray-pyastro16">Python in Astronomy 2016</a></li>
-</ul></p>
-
-<h4><a id="acknowledgements" class="anchor" href="#acknowledgements" aria-hidden="true"><span aria-hidden="true" class="octicon octicon-link"></span></a>Acknowledgements</h4>
```

Comment:
I think the acknowledgements should stay, either here or maybe in the Stingray repository

Reply:
They're listed (albeit without the graphics) in the Stingray docs: https://stingraysoftware.github.io/stingray/acknowledgements.html. @dhuppenkothen indicated she thought that was the best place for them. Should I add the logos there as well?

Reply:
Ah, ok. I think that Stingraysoftware benefited as a whole from PyCharm's free licenses and from GSoC, so it's maybe good to have them here too. @dhuppenkothen, @abigailStev, what do you think?